src/Exclusion/Check/ExcludeByDocBlock.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Exclusion\Check;
+
+use PhpParser\Comment\Doc;
+use PhpParser\Node;
+use PhpParser\Node\Stmt;
+use Rector\Contract\Exclusion\ExclusionCheckInterface;
+use Rector\Contract\Rector\PhpRectorInterface;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+
+final class ExcludeByDocBlock implements ExclusionCheckInterface
+{
+    public function shouldExcludeRector(PhpRectorInterface $phpRector, Node $node): bool
```

Comment:
Better name would be covering both arguments, `shouldExcludeRectorAndNode()`, or better "isNodeSkippedByRector()".

*Exclude* is rather "remove completelly" and is already used, see README: https://github.com/rectorphp/rector#exclude-paths-and-rectors

*Skip* is better - "just this once"

Reply:
fixed